src/App.tsx

```diff
@@ -1,110 +1,34 @@
-import React from 'react';
+import React, { useState } from 'react';
 import './App.css';
 import { getNumbers } from './utils';
+import { Pagination } from './components/Pagination';
+import { DropDown } from './components/DropDown/DropDown';
 
 // eslint-disable-next-line @typescript-eslint/no-unused-vars
 const items = getNumbers(1, 42).map(n => `Item ${n}`);
 
 export const App: React.FC = () => {
+  const [perPage, setPerPage] = useState(5);
+  const [page, setPage] = useState(1);
+  const [elements, setElements] = useState<string[]>([]);
```

Comment:
The `elements` state is initialized as an empty array and never updated within this component directly. Ensure that it is being properly set within child components or consider removing it if it's not used.